### ISSUE TITLE
Handeling calendar 365 [or non-standard]

### ISSUE THREAD

**Issue by ShervanGharari** [ShervanGharari/EASYMORE#49]
EASYMORE nc_remapper() may not work with non-standard calendars such as 365. This will be fixed in the next version.
Meanwhile, you can use local installation by following the steps below.
```
git clone https://github.com/ShervanGharari/EASYMORE.git
cd EASYMORE/easymore
```
in EASYMORE/easymore folder replace line 1328 of easymore.py with (make sure with the right indentation as the current code):
```
ds_temp = ds.isel(time=m)
```
Finally, install locally making sure the existing version is uninstalled
```
pip uninstall easymore
pip install path/to/where/EASYMORE/is/cloned/.
```

**Comment by ShervanGharari** [comment]
The source code, version 1.1.0, on the `main` branch, should work with calendar 365.
Local installation is needed.
In the near future, version 1.1.0 will be released on PyPI.

**Comment by ShervanGharari** [comment]
This should have been resolved in version 2.0.0 on the `main` branch. an example of this case is added here in the example folder.
